gnoi/os/client_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"google.golang.org/grpc"
 )
 
+const readChunkSize = 4000000
```

Comment:
nit: Add a comment, for the units and background as to why this is a reasonable value to chose.